enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+---
+
+# Automatic Updates
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Operator-managed updates are [one of the key benefits of OpenShift 4][architecture].
+Cluster update suggestions are distributed via Cincinnati][cincinnati-openshift], so clusters only receive update recommendations that are appropriate for their cluster.
+While manually approving updates at a per-cluster level is possible, it should not be required.
+This enhancement adds a new `automaticUpdates` property to [the ClusterVersion `spec`][api-spec] so cluster adminstrators can opt in to or out of automatic updates.
+
+## Motivation
+
+Manually approving updates is tedious, doesn't scale well for administrators managing many clusters, and delays the application of potential security fixes.
+Administrators may wish to enable automatic updates for any of those or other reasons.
+Currently, administrators must implement custom polling logic to check for and apply any available updates.
+This enhancement would provide those administrators with a convenient property instead.
+
+### Goals
+
+* Make it easy to opt in to and out of automatic updates.
+
+### Non-Goals
+
+* Support calendar gating or other logic about when updates may be attempted.
+* Choose whether the installer defaults to enabling automatic updates or not for new clusters.
+
+## Proposal
+
+As proposed in [this API pull request][api-pull-request], to add a new property:
+
+```go
+// automaticUpdates enables automatic updates.
+// +optional
+AutomaticUpdates bool `json:"automaticUpdates,omitempty"`
+```
+
+to [`ClusterVersionSpec`][api-spec] to record the administrator's preference.
+
+### User Stories
+
+#### Canary Clusters
+
+Alice has a canary clusters she wants to update as soon as an updated release is available in her configured channel.
+Before this enhancement she would need to construct a poller to run `oc adm upgrade --to-latest` or similar.
+With this enhancement, she can set `automaticUpdates` and does not need to write an external poller.
+
+### Risks and Mitigations
+
+Cluster-managed updates are a relatively new thing, and while we have dozens of OpenShift 4 releases out so far, we have had a few updates result in stuck clusters (like [this][machine-config-operator-drain-wedge].
+Allowing automatic updates would make it more likely that upgrade failures happen when there is no admistrator actively watching to notice and recover from the failure.
+This is mitigated by:
+
+* Alerting.
+    The cluster should fire alerts (FIXME: which ones?) when an upgrade gets into trouble.
+    Adminstrators can configure the cluster to push those alerts out to the on-call admistrator to recover the cluster.
```

Comment:
s/Adminstrators/Administrators/
s/admistrator/administrator/

(I think your spellchecker has some issues)

Reply:
> s/Adminstrators/Administrators/
> s/admistrator/administrator/

Fixed in dae7d51 -> 9b61a96.